module-4/backend/week_one.md

```diff
@@ -1,21 +1,71 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+>That I can make the connection between Ruby and JS and apply those changes pretty well.
+
 2. What are some tools to help debug JavaScript code?
+
+>Debugger, console.log(), breakpoints
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+>Mocha, Chai, npm, node modules
+
 4. What is the syntax for invoking a function?
+
+>functionName()
+
 5. What is CORS?
```

Comment:
5 & 6 are questions from an old version of the repo. Be sure to pull the latest version